spynnaker8/__init__.py

```diff
@@ -580,7 +580,7 @@ def get_time_step():
     """
     if not globals_variables.has_simulator():
         raise ConfigurationException(FAILED_STATE_MSG)
-    return float(__pynn["get_time_step"]()) / 1000.0
+    return float(__pynn["get_time_step"]())
```

Comment:
This linking with PyNN is a bit miserable; I assume this then calls the dt function?

Reply:
The linking with Pynn is even more miserable when PyNN then changes the method name in the passthrough to the simulator.

Yes as shown in https://github.com/SpiNNakerManchester/sPyNNaker8/issues/313 that pynn miserableness calls the dt function